stylus/components/forms.styl

```diff
@@ -467,6 +467,49 @@ $field-inline
         flex-direction column
         margin-left 0
 
+$inputgroup
+    display inline-flex
+    flex-direction row
+    align-items stretch
+    width 100%
+    max-width rem(512)
```

Comment:
I am not sure we want to set a max-width here. It seems like an arbitrary limit.

Reply:
This is the same `max-width` the `input` has.

Reply:
A shared variable between input and inputgroup would be more explicit IMHO :)